modules/installation-aws-arm-tested-machine-types.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// aws modules
+
+[id="installation-aws-arm-tested-machine-types_{context}"]
+= Tested instance types for AWS ARM
```

Comment:
the original title is `Supported AWS machine types`, now we changed it to `Test ...` , my concern is that, users might ask "why isn't the supported list?"

@cuppett @patrickdillon PTAL, thanks.